### PR TITLE
[FLOC-3962] Render acceptance.yml as a template so newlines are preserved.

### PR DESCRIPTION
**Pull request by sarum90**
Prior to this commit, jinja2 was implicitly rendering acceptance.config as 
_something_, and this something could be parsed by the python yaml library.
This worked fine for everything _except_ for the new private key that is going
to be added in order to do authentication with GCE.

Using the template ansible command to render the jinja2 directly so that we get to keep the whitespace in the yaml file. (It seems that specifying content directly in the ansible recipe results in the whitespace being eaten by some different layer of yaml parsing)

### CODE REVIEW

File: roles/local.Azulinho.azulinho-jenkins-server/tasks/main.yaml [hunk 1]
```diff
@@ -164,7 +164,11 @@
   tags: ['jenkins', 'jenkins_config']
 
 - name: Copy config contents of acceptance.yaml to config dir
-  copy: content="{{ acceptance.config }}"
+  # Jinja is nice enough to render our accpetance yaml blob as JSON, but we
```
Comment (Azulinho) [inline roles/local.Azulinho.azulinho-jenkins-server/tasks/main.yaml]: 'accpetance'

File: roles/local.Azulinho.azulinho-jenkins-server/tasks/main.yaml [hunk 2]
```diff
@@ -164,7 +164,11 @@
...
+  # have to double escape backslashes so that when this is evaluated as yaml
+  # (or perhaps for when the string is evaluated by the ansible command
+  # processor) they are preserved.
+  copy: content='{{ acceptance["config"] | to_json | replace("\\", "\\\\")}}'
     dest=/etc/slave_config/acceptance.yaml
   tags: ['jenkins', 'jenkins_config']
```
Comment (Azulinho) [inline roles/local.Azulinho.azulinho-jenkins-server/tasks/main.yaml]: I think you may be able to pass this as a template instead,
```
template: 
    src=etc.slave_config.acceptance.yaml.j2 
    dest=/etc/slave_config/acceptance.yaml 
```
where the roles/local.Azulinho.azulinho-jenkins-server/templates/acceptance.j2 file:
```
{{ acceptance["config"] }}
```

**Comment by Azulinho** [comment]
just some recommendations, feel free to choose to implement them or not.

my only blocker is that, I would like to make sure there is a full test build of flocker on your jenkins test box with these changes applied.
Just to make sure the | to_json | strip_stuff doesn't hurt anything else

**Comment by sarum90** [comment]
Using new format, waiting for http://52.26.214.217:8080/job/ClusterHQ-flocker/job/run-acceptance-test-gce-FLOC-3825/ to succeed before merging.

**Comment by sarum90** [comment]
For those following along at home I just updated the description to match the new approach.

**Comment by sarum90** [comment]
Looks like I didn't break anything :) merging presently then re-deploying jenkins.

**Pull request merged**